hubmap/query_app/filters.py

```diff
@@ -266,7 +272,7 @@ def get_cluster_filter(query_params: dict):
 
         cell_qs = Cell.objects.filter(cell_id__in=input_set)
 
-        cluster_pks = [cluster for cell in cell_qs for cluster in cell.clusters]
+        cluster_pks = [cluster.id for cell in cell_qs for cluster in cell.clusters.all()]
```

Comment:
Will this list contain duplicates of each cluster PK? If so, could that affect query performance -- would it be worthwhile to switch this to a set comprehension?